lang/ilias_en.lang

```diff
@@ -16302,6 +16302,8 @@ tbl#:#tbl_template_deleted#:#The saved view has been deleted.
 tbl#:#tbl_templates#:#View
 tos#:#deleteDocument#:#Delete
 tos#:#detachCriterionAssignment#:#Delete
+tos#:#tos_accept_usr_agreement_anonymous#:#Terms of Service
+tos#:#tos_accept_usr_agreement_anonymous_intro#:#Before You proceed to ILIAS You accept the following Terms of Service.
```

Comment:
Shouldn't we use a `you` instead of `You` here?

Reply:
Yes, is changed.